src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -89,6 +92,16 @@
   public static final boolean NUMERIC_PRECISION_MAPPING_DEFAULT = false;
```

Comment:
Might want to add how this is related to `numeric.mapping`, namely that `numeric.precision.mapping=true` is identical to `numeric.mapping=precision_only` but that `numeric.mapping=best_fit` is often a superior choice since it correctly maps numerics to integral and floating point numbers.

Reply:
I think we can remove this variable. It's only usage is in the ConfigDef below, which has been removed now.

Reply:
Leaving, since we still need to add something to the config definition. Once it's deprecated we can remove all the property specific variables.